plugins/modules/swim_intent.py

```diff
@@ -1028,22 +1042,23 @@ def get_device_ip_from_id(self, device_id):
                 function='get_device_list',
                 params={"id": device_id}
             )
+            self.log("Received API response from 'get_device_list': {0}".format(str(response)), "DEBUG")
             response = response.get('response')[0]
             device_ip = response.get("managementIpAddress")
 
             return device_ip
         except Exception as e:
-            error_message = "Error while getting the response of device from Cisco DNA Center - {0}".format(str(e))
+            error_message = "Error while getting the response of device from Cisco Catalyst Center - {0}".format(str(e))
```

Comment:
as per our coding style start with 
`Exception occurred while getting the ... `

Reply:
changed as per our coding style.